scripts/test_virtualenv.nu

```diff
@@ -0,0 +1,43 @@
+let env_name = 'e-$ èрт🚒♞中片-j'
```

Comment:
Scrambled characters as `env_name`, intended?

Reply:
Yes, it's taken from virtualenv test suite